jOOQ/src/main/java/org/jooq/impl/InsertImpl.java

```diff
@@ -580,6 +580,13 @@ public final InsertImpl defaultValues() {
 
     @Override
     public final InsertImpl doUpdate() {
+        onDuplicateKeyUpdate = true;
+        return this;
+    }
+
+    @Override
+    public final InsertImpl doIgnore() {
+        getDelegate().onDuplicateKeyIgnore(true);
         return this;
```

Comment:
I see, you've done the latter in a separate commit:
https://github.com/jOOQ/jOOQ/pull/5702/commits/8eb788082d57fd34b6cf82c3833e253c86f47ca7